### PR TITLE
[ETCM-411] Add optional param to generate more keys at once

### PR DESCRIPTION
**Pull request by KonradStaniec**
# Description

Add possibility to generate n keys (ported from other project)

### CODE REVIEW

**Review by mmrozek — COMMENTED** [review]
Looks good, but what do you think about moving this functionality to CLI?

**Comment by KonradStaniec** [comment]
one one side it makes sense, on the other side i know that Shay is using it in that format so it would be probably easier for him to use it that way without any learning curve and changes in tooling

**Review by mmrozek — APPROVED** [review]
Ok, so maybe create a task to do that in the future. We could handle both methods at the same time. The cli will be more readable for the users

File: src/main/scala/io/iohk/ethereum/crypto/EcKeyGen.scala [hunk 1]
```diff
@@ -3,19 +3,24 @@ package io.iohk.ethereum.crypto
...
+  * to generate (default is 1).
+  * The key pairs will be printed in the format:
   *   priv-key-hex (32 bytes)
   *   pub-key-hex (64 bytes)
   *
   * Run:
-  *   ./eckeygen > mantis-datadir/node.key
+  *   ./eckeygen [n] > mantis-datadir/node.key
   *
   * to generate the private key for the node. Note that only the private key will be read upon Mantis boot,
```
Comment (mirkoAlic) [inline src/main/scala/io/iohk/ethereum/crypto/EcKeyGen.scala:15]: (minor "Note that only the *first* private key will be read upon Mantis boot"

**Review by mirkoAlic — APPROVED** [review]

File: src/main/scala/io/iohk/ethereum/crypto/EcKeyGen.scala (hunk 1 above)
Comment (KonradStaniec) [reply src/main/scala/io/iohk/ethereum/crypto/EcKeyGen.scala:15]: actually it is more general as even if we generate one pair, only the private key is read at boot time and then whole key pair is regenerated based on this key `keyPairFromPrvKey(Hex.decode(privHex))`

File: src/main/scala/io/iohk/ethereum/cli/CliCommands.scala [hunk 2]
```diff
@@ -27,6 +29,22 @@ object CliCommands {
       }
     }
 
+  private val GenerateKeyPairs: Command[String] =
```
Comment (mmrozek) [inline src/main/scala/io/iohk/ethereum/cli/CliCommands.scala:32]: Please add a test case to `CliCommandsSpec`

**Pull request merged**